public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx

```diff
@@ -43,12 +55,70 @@ export default class ConfigureFieldMapping extends Component<
 
   getAllMappings = async () => {
     this.setState({ loading: true });
-    await this.setState({ mappingsData: EXAMPLE_FIELD_MAPPINGS_RESPONSE });
+    // const mappingsView = await this.props.filedMappingService.getMappingsView(
```

Comment:
Commented code, should it be removed?

Reply:
good catch, must have changed this for local testing, let me revert this

Reply:
fixed